Makefile

```diff
@@ -159,6 +162,38 @@ check-requirements: requirements
 	git status -- *requirements.txt */*requirements.txt | grep -q "nothing to commit"
 	@echo "All requirements files up-to-date!"
 
+.PHONY: check-python-packages
+check-python-packages:
+	# Make target which verifies all the components Python packages are valid
+	@echo ""
+	@echo "================== CHECK PYTHON PACKAGES ===================="
+	@echo ""
+
+	test -f $(VIRTUALENV_COMPONENTS_DIR)/bin/activate || virtualenv --python=$(PYTHON_VERSION) --no-site-packages $(VIRTUALENV_COMPONENTS_DIR) --no-download
+
+	# Setup PYTHONPATH in bash activate script...
+	# Delete existing entries (if any)
+	sed -i '/_OLD_PYTHONPATHp/d' $(VIRTUALENV_COMPONENTS_DIR)/bin/activate
+	sed -i '/PYTHONPATH=/d' $(VIRTUALENV_COMPONENTS_DIR)/bin/activate
+	sed -i '/export PYTHONPATH/d' $(VIRTUALENV_COMPONENTS_DIR)/bin/activate
+
+	echo '_OLD_PYTHONPATH=$$PYTHONPATH' >> $(VIRTUALENV_COMPONENTS_DIR)/bin/activate
+	echo 'PYTHONPATH=${ROOT_DIR}:$(COMPONENT_PYTHONPATH)' >> $(VIRTUALENV_COMPONENTS_DIR)/bin/activate
+	echo 'export PYTHONPATH' >> $(VIRTUALENV_ST2CLIENT_DIR)/bin/activate
+	touch $(VIRTUALENV_COMPONENTS_DIR)/bin/activate
+	chmod +x $(VIRTUALENV_COMPONENTS_DIR)/bin/activate
+	
+	@for component in $(COMPONENTS_WITHOUT_ST2TESTS); do \
```

Comment:
The only complain that iterating over all services includes st2client, which duplicates `.st2client-install-check` that already ran during the `make ci-checks`